BTCPayServer.Common/Altcoins/BTCPayNetworkProvider.BPlus.cs

```diff
@@ -5,22 +5,22 @@ namespace BTCPayServer
 {
     public partial class BTCPayNetworkProvider
     {
-        public void InitBitcoinplus()
+        public void InitBPlus()
         {
             var nbxplorerNetwork = NBXplorerNetworkProvider.GetFromCryptoCode("XBC");
             Add(new BTCPayNetwork()
             {
                 CryptoCode = nbxplorerNetwork.CryptoCode,
-                DisplayName = "Bitcoinplus",
+                DisplayName = "BPlus",
                 BlockExplorerLink = NetworkType == NetworkType.Mainnet ? "https://chainz.cryptoid.info/xbc/tx.dws?{0}" : "https://chainz.cryptoid.info/xbc/tx.dws?{0}",
                 NBXplorerNetwork = nbxplorerNetwork,
-                UriScheme = "bitcoinplus",
+                UriScheme = "bplus-fix-it",
```

Comment:
```suggestion
                UriScheme = "bitcoinplus",
```

just their uri scheme, let it be

Reply:
Apparently it is not even registered and used, so resolving this